parametermanager/api/ParameterManager.Samples.Tests/ParameterManagerFixture.cs

```diff
@@ -73,6 +81,21 @@ public string RandomId()
         return (parameterId, versionId, parameter, payload);
```

Comment:
I know this is not added on this PR, but what is this method for, `CreateParameterWithVersion`? It's really not doing that, it's just creating a parameter and returning some versionId and payload. Can you delete this method?